.github/workflows/build-deploy-to-environment.yml

```
@@ -0,0 +1,93 @@
+name: Build and publish to a specific environments
```

```suggestion
name: Manually build and publish to a specific environments
```

also: probably rename the file to something similar. `manual-build-deploy-to-env` for instance?